cmd/cmd.go

```diff
@@ -908,8 +902,27 @@ func checkServerHeartbeat(_ *cobra.Command, _ []string) error {
 	return nil
 }
 
+func versionHandler(cmd *cobra.Command, _ []string) {
+	fmt.Printf("ollama version %s\n", version.Version)
+
+	client, err := api.ClientFromEnvironment()
+	if err != nil {
+		return
+	}
+
+	serverVersion, err := client.Version(cmd.Context())
+	if err != nil {
+		return
+	}
+
+	if serverVersion != version.Version {
```

Comment:
I think it's great to only show the single version unless there's a mismatch, but I think this should probably be more english text. It should look like:
```
Ollama version: 0.1.2
Warning: Your client is using version 0.1.3
```